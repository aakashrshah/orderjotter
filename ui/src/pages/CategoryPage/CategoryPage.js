import React from 'react';
import { connect } from 'react-redux';
import Grid from '@material-ui/core/Grid';

export function CategoryPage() {
  return (
    <Grid container spacing={1}>
      <Grid container spacing={1} direction="row" />
    </Grid>
  );
}

CategoryPage.propTypes = {
};

export default connect()(CategoryPage);
