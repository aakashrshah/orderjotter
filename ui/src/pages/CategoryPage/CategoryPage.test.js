/* global describe, expect, it */
import React from 'react';
import { shallow } from 'enzyme';

import { CategoryPage } from './CategoryPage';

describe('CategoryPage', () => {
  it('should match a snapshot', () => {
    expect(shallow(<CategoryPage />)).toMatchSnapshot();
  });
});
