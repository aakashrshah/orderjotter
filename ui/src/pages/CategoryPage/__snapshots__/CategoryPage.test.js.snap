// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryPage should match a snapshot 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
  spacing={1}
>
  <WithStyles(ForwardRef(Grid))
    container={true}
    direction="row"
    spacing={1}
  />
</WithStyles(ForwardRef(Grid))>
`;
